src/styles/application/_taskLayout.scss

```diff
@@ -131,6 +131,11 @@
       border-left: var(--pf-global--BorderWidth--sm) solid;
       border-left-color: $pf-color-black-300;
 
+      ul {
```

Comment:
If my comment on the use of `<ul>` versus `<div>` can be utilized, then this will not be necessary.